SWapi-CSharp/Repository.cs

```diff
@@ -122,7 +122,7 @@ public T GetById(int id)
         {
             // TODO: override-able GetPath Method ??
             // TODO: separate UrlBuilderClass
-            string url = this.urlData + this.entity.GetPath() + id;
+            string url = $"{this.urlData}{this.entity.GetPath()}{id}/";
```

Comment:
I think the extra slash is unnecessary.
```suggestion
            string url = $"{this.urlData}{this.entity.GetPath()}{id}";
```